library/aerospike

```diff
@@ -10,8 +10,8 @@ Architectures: amd64
 GitRepo: https://github.com/aerospike/aerospike-server.docker.git
 GitCommit: 41764e219116dcf4a5e1f8107c7cc7e417db32ad
```

Comment:
Any particular reason to keep these old versions listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)